layout/explore/explore-detail/styles.scss

```diff
@@ -5,21 +5,36 @@
         min-height: 100%;
         background-color: $color-12;
 
-        .row {
+        .metadata-section {
             background-color: white;
             margin-bottom: 2 * $space;
             padding: 8 * $space 8 * $space 8 * $space 8 * $space;
-            .column {
-                .title {
-                    margin-top: $space * 4;
-                }
-                h3 {
-                    margin-bottom: 5 * $space;
-                }
-                h4 {
-                    margin: 0px;
-                }
+        }
+        h3 {
+            margin-bottom: 5 * $space;
+        }
+        h4 {
+            margin: 0px;
+        }
+
+        .overview {
+            .title {
+                margin-top: $space * 4;
             }
+            .source-date {
+                display: flex;
+                width: 100%;
+                justify-content: space-between;
+                margin: 2 * $space 0px 5 * $space 0px;
+                font-size: $font-size-extra-small;
+
+                .source {
+                    width: 200px;
```

Comment:
played around with the ellipsis and think we can allow a bit more space! super tiny detail tho

```suggestion
                    width: 80%;
```
Before:
<img width="604" alt="Screenshot 2020-02-25 at 07 07 32" src="https://user-images.githubusercontent.com/971129/75265797-b2663900-57f1-11ea-8ad7-d8608fcef4c2.png">

After: 
<img width="621" alt="Screenshot 2020-02-25 at 07 07 01" src="https://user-images.githubusercontent.com/971129/75265819-ba25dd80-57f1-11ea-9644-307fc9a8a2da.png">